### ISSUE TITLE
Question: Are similar results for QR possible?

### ISSUE THREAD

**Issue by nathanaelbosch** [JuliaLinearAlgebra/RecursiveFactorization.jl#74]
The frontpage results seem quite impressive and this package seems to be the way to go for fast LU factorizations of dense Arrays. Now since I use QR factorizations a lot I wondered: Are similar results possible for QR too? And if yes, are there any plans to make that happen?

**Comment by chriselrod** [comment]
Yes, Yingbo already has an unreleased QR implementation with similarly impressive performance vs OpenBLAS.

**Comment by filtron** [comment]
For what its worth, I would also be interested in this.
